src/services/builds.service.ts

```diff
@@ -42,10 +42,11 @@ async function remove(id: string): Promise<Build> {
   );
 }
 
-async function stop(id: string): Promise<Build> {
+async function update(id: string, body: object): Promise<Build> {
```

Comment:
could we make `body` with specific type instead of `object`?

Reply:
Done.